ironfish/src/metrics/meter.ts

```diff
@@ -22,7 +23,7 @@ export class Meter {
   private _rate5m: RollingAverage
   private _average: RollingAverage
   private _count = 0
-  private _interval: ReturnType<typeof setInterval> | null = null
+  private _interval: SetTimeoutToken | null = null
```

Comment:
Hey, is this compatible with intervals? Should we make a separate SetIntervalToken that is the return type of SetInterval?